### DISCUSSION TITLE
After reimporting the GPU acceleration is gone

### DISCUSSION THREAD

**Discussion by seflerZ** [microsoft/wslg#736] [Q&A] [answered]
I've GPU acceleration enabled Ubuntu22.04 and exported to a tar file to make a backup. When reimported it to WSL2, the OpenGL render string becomes: llvmpipe which is software render. I've tried to reinstall the GPU driver but nothing helps. How to reconfig the GPU accelerations?

**Comment by seflerZ** [comment, marked as answer]
Ok, solved. Looks like the wsl d3d shared lib path has been unexpectly excluded. Adding the "/usr/lib/wsl/lib" path back to "/etc/ld.so.conf" solved this problem.

**Reply by crramirez** [reply]
Good to know
